src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+  let totalCost = days * dayCost;
+
+  if (days >= 3 && days < 7) {
+    totalCost = totalCost - 20;
+  }
+
+  if (days >= 7) {
+    totalCost = totalCost - 50;
```

Comment:
The condition for the second 'if' statement can be simplified. Since the code already returns earlier if 'days' is less than 3, you can remove the lower bound check in the first 'if' statement and remove the second 'if' statement entirely, replacing it with a return statement that applies the appropriate discount based on the number of days.